spec/std/file_spec.cr

```diff
@@ -741,6 +741,17 @@ describe "File" do
     end
   end
 
+  it "reads at offset" do
+    filename = "#{__DIR__}/data/test_file.txt"
+    file = File.open(filename)
+    file.read_at(6, 100) do |io|
+      io.gets_to_end.should eq("World\nHello World\nHello World\nHello World\nHello World\nHello World\nHello World\nHello World\nHello Worl")
+    end
+    file.read_at(0, 240) do |io|
+      io.gets_to_end.should eq(File.read(filename))
+    end
+  end
+
```

Comment:
Actually, I see there are such specs for memory. Maybe add them to file as well?